lib/backends/vault-backend.js

```diff
@@ -55,10 +57,16 @@ class VaultBackend extends KVBackend {
     const secretResponse = await this._client.read(key)
 
     if (kvVersion === 1) {
+      if (keyOptions && keyOptions.isBinary) {
+        secretResponse.data[property] = Buffer.from(secretResponse.data[property], 'base64').toString('utf8')
```

Comment:
Would it be enough to not do the `toString('utf8')` here, just converting the base64 string to a buffer. later on this buffer will be base64 encoded again as the base class won't mess with buffers. This should put the original value from vault into the secret without it ending up double base64 encoded.

https://github.com/godaddy/kubernetes-external-secrets/blob/c3c27bc427c7129e4e98b5b10e4c5dde146c422d/lib/backends/kv-backend.js#L91-L93

https://github.com/godaddy/kubernetes-external-secrets/blob/c3c27bc427c7129e4e98b5b10e4c5dde146c422d/lib/backends/kv-backend.js#L130